Tabloid/client/src/Managers/TagManager.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
```

Comment:
Import React not needed, remove line to get rid of browser console warning message.

Reply:
fixed